interactive_markers/interactive_marker_server.py

```diff
@@ -138,9 +138,14 @@ def shutdown(self):
         can be destroyed.
         """
         self.clear()
```

Comment:
@jacobperron nit: would it make sense to put `self.clear()` right before `self.applyChanges()`, within the `if` clause? If we're not propagating changes, there's no point in clearing.